recipes/refinitiv-dataplatform/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "refinitiv-dataplatform" %}
+{% set version = "1.0.0a0" %}
+{% set sha256 = "9f192b39db20a3e07b1f89ecbb2aea2015351edad0561d7d139c8d64562bec9d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - python-dateutil
+    - deprecation
+    - appdirs >=1.4.3
+    - requests >=2.21.0
+    - requests-async >=0.6.2
+    - nest-asyncio >=1.0.0
+    - websocket-client >=0.54.0
+    - pandas >=0.17.0
+    - numpy >=1.11.0
+    
+test:
+  imports:
+    - refinitiv.dataplatform
+
+about:
+  home: https://developers.refinitiv.com/
+  license: Apache-2.0
```

Comment:
Where is this licence coming from? The PyPI just shows `License: LICENSE`.

Reply:
@chrisburr I use the same as Eikon library. 
However, I will ask the team who published package to add license on PyPI.  
I might take a bit long because right now the team is on holiday. They will be back after New Year.
If it's completed, I will ask you to review again.

Thanks